src/Extension/CoreExtension.php

```diff
@@ -1285,6 +1287,75 @@ public static function testEmpty($value): bool
         return '' === $value || false === $value || null === $value || [] === $value;
     }
 
+    /**
+     * Checks if a variable is traversable.
+     *
+     *    {# evaluates to true if the foo variable is an array or a traversable object #}
+     *    {% if foo is iterable %}
+     *        {# ... #}
+     *    {% endif %}
+     *
+     * @param mixed $value A variable
+     *
+     * @return bool true if the value is traversable
+     *
+     * @deprecated since Twig 3.8, to be removed in 4.0 (use the native "is_iterable" function instead)
+     *
+     * @internal
+     */
+    public static function testIterable($value)
+    {
+        return is_iterable($value);
+    }
```

Comment:
This method should be removed, right?

Reply:
I would keep `iterable` test in Twig, because:

- `mapping` & `sequence` tests are not replacing it. Jinja has all three: https://jinja.palletsprojects.com/en/3.0.x/templates/#builtin-tests
- it also cover strings AFAIK, so `mapping` + `sequence` != `iterable`
- removing it will introduce a compatibility break

Reply:
the comment is not about removing the Twig test. It is about removing the internal method used to implement it, as `is_iterable` can be used directly as the implementation of the Twig test (and it is already used directly btw).

Reply:
ok, got it. I would do the change in the next commit.

Reply:
well, introducing a new method that is already deprecated does not make sense. A deprecation is meant to avoid breaking existing code that uses that method, delaying the removal until the next major version where BC breaks are allowed.
For a new method, there is no existing code that could break, as the method does not exist at all in previous versions

Reply:
OK, understood. I t may be the leftover of a merge. Removed in https://github.com/twigphp/Twig/pull/3859/commits/db3753d8464804c96b067dff8e7619152b21ad33